.circleci/config.yml

```diff
@@ -332,6 +332,22 @@ jobs:
       - checkout
       - run: ./tools/ci/init_gcloud
       - run: ./tools/ci/report_coverage
+  prepublish:
```

Comment:
How do you feel about calling this `prepublish_npm`?

Reply:
sure